specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/examples/Function_Delete.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parameters": {
+    "api-version": "2017-04-01-preview",
+    "subscriptionId": "56b5e0a9-b645-407d-99b0-c64f86013e3d",
+    "resourceGroupName": "sjrg1637",
```

Comment:
Is there special reason on using diferent resourceGroupName on different examples?
To make the examples available for generating test, they should use one unique resource group

Reply:
Hi @changlong-liu  thank you for bring it up, I think there are no special reason on using different resourceGroupName. That may just be copied from real request/response. 

I get context from email thread that you are working on helping ASA to auto-generate CLI based on swagger files, right? 

Do we just need to update  resourceGroupName to using one unique value? Are there any other issues may block generating CLI?

Reply:
Updated